internal/archive/module_location.go

```diff
@@ -1,27 +1,23 @@
 package dir
 
-// ModuleLoc - module location type that provides services for stand along module build command
+// ModuleLoc - module location type that provides services for stand alone module build command
 type ModuleLoc struct {
-	loc        *Loc
-	targetPath string
+	loc *Loc
 }
 
 // GetTarget - gets the target path;
 // if it is not provided, use the path of the processed project.
 func (ep *ModuleLoc) GetTarget() string {
-	return ep.targetPath
+	return ep.loc.GetTarget()
 }
 
 // GetTargetTmpDir - gets the temporary target directory path.
 // The subdirectory in the target folder is named as the source project folder suffixed with "_mta_build_tmp".
 // Subdirectory name is prefixed with "." as a hidden folder
-// in case of stand along module build when target folder provided build results will be saved in this target folder
+// in case of stand alone module build when target folder provided build results will be saved in this target folder
```

Comment:
There is no target folder to provide anymore, this comment should be changed accordingly (also it's always saved in the target folder and never in the temp directory)

Reply:
changed